src/HomeHeader.jsx

```diff
@@ -0,0 +1,67 @@
+import "./homeHeader.css"
+import Logo from "./static/images/logo.jpg"
+import Restaurant from "./static/images/restaurant.png"
+import ArrowDown from "./static/images/arrow-down.svg"
+import Search from "./static/images/search.svg"
+import Pin from "./static/images/pin-o.svg"
+import Promotion from "./static/images/promotion.svg"
+import User from "./static/images/user.svg"
+import Bag from "./static/images/bag.svg"
+
+function HomeHeader(){
+  return(
+    <header className="home_header">
+        <nav class="home-header__container">
+            <div class="home-header__logo">
+              <img src={Logo} width="100" height="100" />
+            </div>
+            <section class="multi-categories-selector__label">
+                <div class="multi-categories-selector__background-colors-restaurant">
+                    <img src={Restaurant} width="40" height="40" />
+                </div>
+                <div class="multi-categories-selector__label-title">
+                    <span>Restaurante</span>
+                </div>
+                <div class="multi-categories-selector__label-arrow">
+                    <span class="icon-marmita--arrow-down">
+                        <img src={ArrowDown} />
+                    </span>
+                </div>
+            </section>
+            <section class="search-container">
+                <form class="search-input__form">
+                    <div class="search-input__form-wrapper">
+                        <img src={Search} class="search_image" />
+                        <input class="search-input-field" tabindex="0" role="search" type="text" autocomplete="off"
+                            aria-label="Busque por item ou loja" 
+                            placeholder="Busque por item ou loja" value="" />
```

Comment:
you can remove this value="", because it's not possible to fill in the input if you set a value empty.

I removed it here, and now it's possible:
![image](https://user-images.githubusercontent.com/38358239/150129645-98b7072d-4b20-4792-bed9-2ecd627e2e86.png)

Reply:
Tks rai, removed 👀